dotCMS/src/main/java/com/dotcms/rendering/velocity/directive/FixBreaks.java

```diff
@@ -30,8 +30,6 @@ public String getName() {
     return "fixBreaks";
   }
 
-
-
   final public boolean render(InternalContextAdapter context, Writer writer, Node node)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)